pkg/scheduler/internal/cache/debugger/comparer.go

```diff
@@ -57,11 +57,11 @@ func (c *CacheComparer) Compare() error {
 	pendingPods := c.PodQueue.PendingPods()
 
 	if missed, redundant := c.CompareNodes(nodes, dump.Nodes); len(missed)+len(redundant) != 0 {
-		klog.Warningf("cache mismatch: missed nodes: %s; redundant nodes: %s", missed, redundant)
+		klog.InfoS("Cache mismatch", "missedNodes", missed, "redundantNodes", redundant)
```

Comment:
ditto, keep it as "missed nodes" and "redundant nodes", below too.

Reply:
+1

Reply:
if K has spaces, will it be confused?
eg:
```
"Cache mismatch" missed nodes=[node1 test1] redundant nodes=[node2 test2]
```
I will keep it as "missed nodes" and "redundant nodes". Thanks @ahg-g @chendave